ansible_events_ui/api/activation.py

```diff
@@ -1,8 +1,125 @@
 """Activation API endpoints."""
-from fastapi import APIRouter
+import logging
+import json
+
+from fastapi import APIRouter, Depends
+import asyncio
+
+import sqlalchemy.orm
+from sqlalchemy import insert, select
```

Comment:
Minor: Since select and insert terms are quite generic, it's slightly better to use module import:

```
import sqlalchemy as sa
```